Assets/Scripts/Enemy/Enemy.cs

```diff
@@ -22,7 +22,7 @@ void Awake()
         if (playerBase == null)
             playerBase = BaseController.Singleton.transform;
 
-        GetComponent<HealthLogic>().onDeath += Die;
+        //GetComponent<HealthLogic>().onDeath += Die;
```

Comment:
Why was this (and the corresponding line in `OnDestroy()` below) commented out? 🤔